ta_services/keymaster/entry.c

```diff
@@ -44,7 +58,14 @@ TEE_Result TA_InvokeCommandEntryPoint(void __unused *sess, uint32_t cmd,
 				      uint32_t pt,
 				      TEE_Param params[TEE_NUM_PARAMS])
 {
+	if (cmd != KEYMASTER_CMD_CONFIGURE && !version_info_set) {
```

Comment:
I'd prefer if the `km_` family of functions kept track of when they can be called and not themselves. `entry.c` is suppose to be just a translation layer, not an enforcer of policy etc.

Reply:
Ok. Will move to `km.c`.